src/locale/en.json

```
@@ -816,6 +816,9 @@
     "Unknown": "Challenge",
     "Weekly": "Weekly Challenge"
   },
+  "Mods": {
+    "HarmonicModDescription": "This mod's effect comes at a reduced cost and changes depending on the subclass used."
```

```suggestion
    "HarmonicModDescription": "This mod's effect comes at a reduced cost and changes element depending on the equipped subclass."
```
apologies, meant to change this one too